cluster/gce/upgrade.sh

```diff
@@ -552,7 +552,7 @@ function update-coredns-config() {
 }
 
 echo "Fetching the previously installed CoreDNS version"
-CURRENT_COREDNS_VERSION=$("${KUBE_ROOT}/cluster/kubectl.sh" -n kube-system get deployment coredns -o=jsonpath='{$.spec.template.spec.containers[:1].image}' | cut -d ":" -f 2)
+CURRENT_COREDNS_VERSION=$("${KUBE_ROOT}/cluster/kubectl.sh" -n kube-system get deployment coredns -o=jsonpath='{$.spec.template.spec.containers[:1].image}' | sed -r 's/.+:v?(.+)/\1/'
```

Comment:
I think it would be more resilient if the cut command stayed and optionally chopping off the v prefix.

Reply:
same with above

Reply:
The current version would be without the v prefix (k8s.gcr.io/coredns:1.7.0)and the new version would be with the v prefix (k8s.gcr.io/coredns/coredns:v1.8.0). I think sed would cover both cases.